modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractKotlinCodegen.java

```diff
@@ -453,13 +453,22 @@ public String toModelName(final String name) {
         String modifiedName = name.replaceAll("\\.", "");
         modifiedName = sanitizeKotlinSpecificNames(modifiedName);
 
+        // Camelize name of nested properties
+        modifiedName = camelize(modifiedName);
```

Comment:
I think this is fine because it follows [kotlin naming conventions](https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules). However, this will potentially break for anyone using only the models of kotlin-client for non-JSON serialization that relies on model names for serialization, which I believe is what I was doing when I original wrote the kotlin client.